src/rust/engine/src/core.rs

```diff
@@ -60,11 +60,6 @@ pub struct TypeConstraint(pub Id);
 #[derive(Clone, Copy, Debug, Default, Eq, Hash, PartialEq)]
 pub struct Function(pub Id);
 
-// The name of a field.
```

Comment:
Yeah. I meant to ask whether you'd prefer that. I think having the alias clarifies the purpose, I'll add it back in.